src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/adjudications/AdjudicationsADAAWardSummaryResourceIntTest.kt

```diff
@@ -0,0 +1,233 @@
+import uk.gov.justice.digital.hmpps.nomisprisonerapi.integration.IntegrationTestBase
+import uk.gov.justice.digital.hmpps.nomisprisonerapi.jpa.AdjudicationHearing
+import uk.gov.justice.digital.hmpps.nomisprisonerapi.jpa.AdjudicationHearingResult
+import uk.gov.justice.digital.hmpps.nomisprisonerapi.jpa.AdjudicationHearingType
+import uk.gov.justice.digital.hmpps.nomisprisonerapi.jpa.AdjudicationIncident
+import uk.gov.justice.digital.hmpps.nomisprisonerapi.jpa.AdjudicationIncidentCharge
+import uk.gov.justice.digital.hmpps.nomisprisonerapi.jpa.AgencyInternalLocation
+import uk.gov.justice.digital.hmpps.nomisprisonerapi.jpa.Offender
+import uk.gov.justice.digital.hmpps.nomisprisonerapi.jpa.Staff
+import java.time.LocalDate
+import java.time.LocalDateTime
+
+class AdjudicationsADAAWardSummaryResourceIntTest : IntegrationTestBase() {
+  @Autowired
+  lateinit var repository: Repository
+  lateinit var aLocationInMoorland: AgencyInternalLocation
+
+  @Autowired
+  private lateinit var nomisDataBuilder: NomisDataBuilder
+
+  @BeforeEach
+  fun setUp() {
+    aLocationInMoorland = repository.getInternalLocationByDescription("MDI-1-1-001", "MDI")
+  }
+
+  @DisplayName("GET /prisoners/booking-id/{bookingId}/awards/ada/summary")
+  @Nested
+  inner class GetAwardsAdaSummary {
+    private val offenderNo = "A1965NM"
+    private var bookingId = 0L
+    private lateinit var prisoner: Offender
+    private lateinit var reportingStaff: Staff
+    private lateinit var existingIncident: AdjudicationIncident
+    private lateinit var previousIncident: AdjudicationIncident
+    private val existingAdjudicationNumber = 123456L
+    private val previousAdjudicationNumber = 123455L
+    private lateinit var existingHearing: AdjudicationHearing
+    private lateinit var existingCharge: AdjudicationIncidentCharge
+    private lateinit var existingHearingResult: AdjudicationHearingResult
+    private lateinit var previousHearingResult: AdjudicationHearingResult
+
+    @BeforeEach
+    fun createPrisonerWithAdjudicationAndHearingAndAwards() {
+      nomisDataBuilder.build {
+        reportingStaff = staff(firstName = "JANE", lastName = "STAFF") {
+          account(username = "JANESTAFF")
+        }
+        existingIncident = adjudicationIncident(reportingStaff = reportingStaff) {}
+        previousIncident = adjudicationIncident(reportingStaff = reportingStaff) {}
+        prisoner = offender(nomsId = offenderNo) {
+          bookingId = booking(agencyLocationId = "BXI", bookingBeginDate = LocalDateTime.parse("2022-01-01T10:00")) {
+            prisonTransfer(from = "BXI", to = "MDI", date = LocalDateTime.parse("2022-01-02T10:00"))
+            prisonTransfer(from = "MDI", to = "BXI", date = LocalDateTime.parse("2022-01-03T10:00"))
+            prisonTransfer(from = "BXI", to = "BMI", date = LocalDateTime.parse("2022-01-03T10:00"))
+            adjudicationParty(incident = previousIncident, adjudicationNumber = previousAdjudicationNumber) {
```

Comment:
I will also add a test when there are no transfers (knowing for well it fails since new bookings don't create a external movement record in the builder yet - but should do)